annotate.el

```diff
@@ -730,6 +737,25 @@ file."
 file."
   (nth 1 record))
 
+(defun annotate-filename-from-dump (record)
+  "Get the filename field from an annotation list loaded from a
+file."
+  (cl-first record))
+
+(defun annotate-start-annotation-dump (annotation)
```

Comment:
"start" sounds like a verb. Similarly to the previous function, I would propose something like `annotate-beginning-of-annotation`.

Reply:
OK!